latte-dock: add page (#8819)
tldr-pages/tldr: pages/linux/latte-dock.md

# latte-dock

> > Replacement dock for Plasma desktop with an elegant and intuitive experience for your tasks and plasmoids.

Reviewer: We prefer an objective command description
```suggestion
> Replacement dock for Plasma desktop.
```

> > Replacement dock for Plasma desktops, providing an elegant and intuitive experience for your tasks and plasmoids.

Reviewer 2: ```suggestion
> Replacement dock for Plasma desktop with an elegant and intuitive experience for your tasks and plasmoids.
```
Even though the main description mentions `desktops`, I think `desktop` is correct here and it is a typo there. This can be referred to as you read their README file.

> More information: <https://github.com/KDE/latte-dock>.

> - Clear qml cache:

Reviewer: ```suggestion
- Clear QML cache:
```

> `latte-dock --cc`

Reviewer: We prefer long arguments if available
```suggestion
`latte-dock --clear-cache`
```

- Import and load default layout on startup:

`latte-dock --default-layout`

> - Load specific layout on startup:

Reviewer: ```suggestion
- Load a specific layout on startup:
```

`latte-dock --layout {{layout_name}}`

> - Import and load a layout:

Reviewer: ```suggestion
- Import and load a specific layout:
```

> `latte-dock --import-layout  {{path/to/file}}`

Reviewer: ```suggestion
`latte-dock --import-layout {{path/to/file}}`
```
